tests/mobly/controllers/android_device_lib/fastboot_test.py

```diff
@@ -161,6 +161,38 @@ def test_fastboot_args_with_custom_timeout(self, mock_run_command):
         timeout=20,
     )
 
+  @mock.patch('mobly.utils.run_command')
+  def test_fastboot_exe_cmd_without_timeout_arg(self, mock_run_command):
+    expected_stdout = 'stdout'
+    expected_stderr = b'stderr'
+    mock_run_command.return_value = (123, expected_stdout, expected_stderr)
+
+    fastboot.exe_cmd('fastboot -w')
```

Comment:
does this test fail without the fix?

Reply:
Yes, this test will fail bc of missing parameter `timeout` in call to function `exe_cmd`.